packages/ag-charts-website/src/content/docs/axes-ticks/_examples/axis-tick-interval/main.ts

```diff
@@ -28,21 +28,19 @@ const options: AgCartesianChartOptions = {
             title: {
                 text: 'Market Share (%)',
             },
-            tick: {
-                interval: 20,
-            },
+            interval: 20,
         },
     ],
 };
 
 const chart = AgCharts.create(options);
 
 function setTickInterval(interval: number) {
-    options.axes![1].tick = { interval };
+    Object.assign(options.axes![1], { interval });
```

Comment:
Nit: Maybe `options.axes![1].interval = interval`?

Reply:
The problem with all these cases was it required a specific type to pass TS checks, so the alternative is actually:
`(options.axes![1] as AgContinuousAxisOptions).interval = interval`